### PR TITLE
COMMERCE-9299

### PR DESCRIPTION
**Pull request by ghost**
https://issues.liferay.com/browse/COMMERCE-9299

### CODE REVIEW

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-cart-test/src/testIntegration/java/com/liferay/headless/commerce/delivery/cart/resource/v1_0/test/AddressResourceTest.java [hunk 1]
```diff
@@ -14,15 +14,275 @@
...
+				_commerceAddress);
+		}
+
+		if (_commerceAccount != null) {
+			_commerceAccountLocalService.deleteCommerceAccount(
+				_commerceAccount);
+		}
+
+		if (_user != null) {
+			UserLocalServiceUtil.deleteUser(_user);
```
Comment (luca-pellizzon) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-cart-test/src/testIntegration/java/com/liferay/headless/commerce/delivery/cart/resource/v1_0/test/AddressResourceTest.java:115]: Don't we trust @DeleteAfterTestRun annotation?
We should use that where possible.
(This is for all the entities used)

**Review by luca-pellizzon — CHANGES_REQUESTED** [review]

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-cart-test/src/testIntegration/java/com/liferay/headless/commerce/delivery/cart/resource/v1_0/test/AddressResourceTest.java (hunk 1 above)
Comment (CrescenzoRegaLF) [reply modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-cart-test/src/testIntegration/java/com/liferay/headless/commerce/delivery/cart/resource/v1_0/test/AddressResourceTest.java:115]: you are right, i will made a changes

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Review by luca-pellizzon — APPROVED** [review]

**Comment by luca-pellizzon** [comment]
ci:forward

**Pull request closed without merging**